tools/onnx-mlir.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/src/Runtime/OMInstrument.inc b/src/Runtime/OMInstrument.inc
```

Comment:
Perhaps we should put this patch inside the 'tools/onnx-mlir/.' directory instead of 'tools/.'

Reply:
Changes present in [3890394](https://github.com/DependableSystemsLab/LLTFI/pull/22/commits/38903946e61169e91056945b7beeadbb756c070f)